backend/plugins/mongolian_api/src/modules/erkhet/graphql/resolvers/queries/remainders.ts

```
@@ -0,0 +1,292 @@
+
+      switch (contentType) {
+        case 'company':
+          const company = await sendTRPCMessage({
+            subdomain,
+            pluginName: 'core',
+            method: 'query',
+            module: 'companies',
+            action: 'findOne',
+            input: { _id: contentId },
+            defaultValue: {},
+          });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)